plugins/modules/rds_cluster.py

```diff
@@ -0,0 +1,1024 @@
+        aliases:
+          - replication_src_id
+        type: str
+    restore_to_time:
+        description:
+          - The UTC date and time to restore the DB cluster to. Must be in the format "2015-03-07T23:45:00Z". If this is not provided while
+            restoring a cluster, I(use_latest_restorable_time) must be. May not be specified if I(restore_type) is copy-on-write.
+        type: str
+    restore_type:
+        description:
+          - The type of restore to be performed. If not provided, Amazon RDS uses full-copy.
+        choices:
+          - full-copy
+          - copy-on-write
+        type: str
+    role_arn:
+        description:
+          - The Amazon Resource Name (ARN) of the IAM role to associate with the Aurora DB cluster, for example
+            "arn:aws:iam::123456789012:role/AuroraAccessRole"
+        type: str
+    s3_bucket_name:
+        description:
+          - The name of the Amazon S3 bucket that contains the data used to create the Amazon Aurora DB cluster.
+        type: str
+    s3_ingestion_role_arn:
+        description:
+          - The Amazon Resource Name (ARN) of the AWS Identity and Access Management (IAM) role that authorizes Amazon RDS to access
+            the Amazon S3 bucket on your behalf.
+        type: str
+    s3_prefix:
+        description:
+          - The prefix for all of the file names that contain the data used to create the Amazon Aurora DB cluster. If you do not
+            specify a SourceS3Prefix value, then the Amazon Aurora DB cluster is created by using all of the files in the Amazon S3 bucket.
+        type: str
+    skip_final_snapshot:
+        description:
+          - Whether a final DB cluster snapshot is created before the DB cluster is deleted. If this is false I(final_snapshot_identifier)
+            must be provided.
+        type: bool
+        default: false
+    snapshot_identifier:
+        description:
+          - The identifier for the DB snapshot or DB cluster snapshot to restore from. You can use either the name or the ARN to specify
+            a DB cluster snapshot. However, you can use only the ARN to specify a DB snapshot.
+        type: str
+    source_db_cluster_identifier:
+        description:
+          - The identifier of the source DB cluster from which to restore.
+        type: str
+    source_engine:
+        description:
+          - The identifier for the database engine that was backed up to create the files stored in the Amazon S3 bucket.
+        choices:
+          - mysql
+        type: str
+    source_engine_version:
+        description:
+          - The version of the database that the backup files were created from.
+        type: str
+    source_region:
+        description:
+          - The ID of the region that contains the source for the DB cluster.
+        type: str
+    storage_encrypted:
+        description:
+          - Whether the DB cluster is encrypted.
+        type: bool
+    allocated_storage:
```

Comment:
hmm do we need this parameter? Because the module allows only aurora engines?

```
        choices:
          - aurora
          - aurora-mysql
          - aurora-postgresql
```

Reply:
Thanks! This param shouldn't be in the options list. I removed it.